src/shared-components/dropdown/test.tsx

```diff
@@ -17,52 +16,48 @@ describe('<Dropdown />', () => {
   describe('on touch screen', () => {
     it('renders <MobileDropdown />', () => {
       window.document.documentElement.ontouchstart = () => undefined;
-      const wrapper = mount(
+      const { getByRole, getAllByRole, queryAllByRole } = render(
         <Dropdown value="test1" options={options} onChange={() => undefined} />,
       );
       delete window.document.documentElement.ontouchstart;
 
-      expect(wrapper.children().children().first().name()).toEqual(
-        'MobileDropdown',
```

Comment:
Both of these tests are, to an extent, testing implementation details. The `@testing-library/react` assertions compare the differences in the markup between Mobile and Desktop, though. The enzyme version is testing the React component name, but not what it actually looks like.